crates/https/src/https_server.rs

```diff
@@ -66,6 +66,7 @@ pub(crate) async fn message_from_post<R>(
 where
     R: Stream<Item = Result<Bytes, h2::Error>> + 'static + Send + Debug + Unpin,
 {
+    #[allow(clippy::min_max)]
```

Comment:
I suppose you missed my comments at https://github.com/bluejekyll/trust-dns/commit/d53381edda7e2c787789b4d5704724cc5f24199b? I think this is wrong.

Reply:
I got it. I have a patch locally...